code/modules/crew_objectives/cargo_objectives.dm

```diff
@@ -76,15 +76,19 @@
 
 /datum/objective/crew/fatstacks/update_explanation_text()
 	. = ..()
-	explanation_text = "Have at least [target_amount] mining points on your ID at the end of the shift."
+	explanation_text = "Have at least [target_amount] mining points on IDs in your possession at the end of the shift."
```

Comment:
It should be based on the account id stored in the mind rather than IDs you hold